src/Symfony/Component/HttpKernel/HttpKernelInterface.php

```diff
@@ -27,7 +27,7 @@ interface HttpKernelInterface
     /**
      * @deprecated Use {@see MAIN_REQUEST} instead
      */
-    public const MASTER_REQUEST = self::MAIN_REQUEST;
+    public const MASTER_REQUEST = DeprecatedConstants::MASTER_REQUEST;
```

Comment:
What happens when using preloading ? Would the deprecated constant be resolved at that point ?